Category-Colors/category-colors
# Add test components

# Description 
Related to Issue #5 
- Attempts to mimic the wireframe using ChakraUI components without any added styling or functionality. 
- Creates a new directory called `components`
- Creates 2 new components (Accordion and Slider) to make things a little cleaner.

# QA
Run the localhost and you should be able to see the below

![Screen Shot 2023-04-23 at 11 06 20 PM](https://user-images.githubusercontent.com/2035194/233891311-cc74724b-1841-499d-ac31-168e8143e1ff.png)
